### ISSUE TITLE
laggy zombies

### ISSUE THREAD

**Issue by caesius90** [skyarkhangel/Hardcore-SK#438]
zombie corpse makes huge lag.

huge amount of zombies appear but no lag 
but after i kill them, it makes huge lag.

i think it comes from saved(contained) information about dead zombies. 
informations like relationship, skill, personality, wounded parts...which is no uses at all...

or reanimating code might be the source of lag...maybe...(but corpse lose thier brain or head still makes lag)

this would be no lagging for high spec computers but low spec computers like mine, this is huge annoying factor for playing this mod pack...

plz investigate this problem. thanks

**Comment by Clownvenus** [comment]
I have an high spec computer and zombies, I lag too when i accelerate the time. I don't have any lag when i started my colony. If you are right, need to built a crematorium for verify that.

**Comment by caesius90** [comment]
clownvenus// yes. removing zombie corpes always resolve the laggy situation. thats the way im using - removing all corpes ASAP

**Comment by Clownvenus** [comment]
Anyway for my next colony, i disable this mod. I have played one year and two seasons with randy random. I only have lot of zombie raid with some events like Insects, idle or raptors but no bandit's raid or other faction.

I suspected this mod reduce the percetage to be attack by humans or i really unlucky. 90 days to kill zombies with arrows in the head :D

And it's possible to cut the head or kill them when they are falling to the ground?
